Directory.Packages.props

```diff
@@ -5,6 +5,7 @@
   <Import Project="eng/Versions.props" Condition="'$(MajorVersion)' == '' and Exists('eng/Versions.props')" />
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
```

Comment:
Nice! didn't know this was a thing.

Reply:
The only thing that makes me a bit worried is that this now may be pinning down transitive dependencies, but I guess that in that case we'd get NuGet warnings, right?

Reply:
I am trying to test this, several Azure SDK libraries requires `Azure.Core >= 1.40.0` (only used as a transitive dependency) so I included `1.39.0` and got errors:

```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error (active)	NU1109	Detected package downgrade: Azure.Core from 1.40.0 to centrally defined 1.38.0. Update the centrally managed package version to a higher version. 
 DatabaseMigration.ApiModel -> Azure.Identity 1.12.0 -> Azure.Core (>= 1.40.0) 
 DatabaseMigration.ApiModel -> Azure.Core (>= 1.38.0)	DatabaseMigration.ApiModel	D:\aspire\playground\DatabaseMigration\DatabaseMigration.ApiModel\DatabaseMigration.ApiModel.csproj	1		
```

Reply:
Ok great, no concerns then.